config/validate.go

```diff
@@ -301,7 +301,7 @@ func (m *Metadata) Validate(data map[string]any) []string {
 					errors = append(errors, msg)
 				} else {
 					fm := mustFloat(validation.Arg)
-					if fv == 0 || fv < fm {
+					if fv != 0 && fv < fm {
```

Comment:
Here's the fix for allowing `minOrZero` validation to pass when a zero value is passed in